src/scripts/configs/embeddables.coffee

```diff
@@ -0,0 +1,41 @@
+define (require) ->
+
+  _ = require('underscore')
```

Comment:
It doesn't appear that underscore is a dependency of this file (the only place I see it being used is commented out).